tests/test_unit/test_io.py

```diff
@@ -0,0 +1,97 @@
+import os
+from pathlib import Path
+
+import h5py
+import numpy as np
+import pandas as pd
+import pytest
+from pydantic import ValidationError
+
+from movement.io import load_poses
+
+
+class TestLoadPoses:
```

Comment:
I think it would be useful to send the logging elsewhere during the tests (e.g. `tmp_path/movement.log`). The users won't be running the tests, but it would be useful to seperate logging during the tests and "normal" use.